src/applications/financial-status-report/containers/App.jsx

```diff
@@ -243,6 +247,7 @@ const mapStateToProps = state => ({
   showWizard: fsrWizardFeatureToggle(state),
   showFSR: fsrFeatureToggle(state),
   showReviewPageNavigationFeature: reviewPageNavigationFeatureToggle(state),
+  isEligibleForStreamlined: isEligibleForStreamlined(state),
```

Comment:
I think we can drop this and the import since w're not using it

Reply:
Great catch, removed!